recipes/ldsc/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -eu -o pipefail
+wget https://patch-diff.githubusercontent.com/raw/bulik/ldsc/pull/176.patch 
```

Comment:
Could you add the 176.patch to the recipe folder instead? In case it's not online anymore some day

Reply:
I'm on it, thanks

Reply:
You can get rid of `wget` in the recipe then :)